layouts/index.html

```diff
@@ -7,9 +7,9 @@
     <a name="page-top"></a>
     {{ partial "nav.html" . }}
 
-    {{ with $.Site.Data.index }}
+    {{ with .Resources.ByType "hero" }}
       {{ range . }}
-        {{ $.Scratch.Set "data" . }}
+        <!-- {{ $.Scratch.Set "data" . }} -->
 
         {{ if and (not .disabled) (eq .fragment "hero") }}
           {{ partial "fragments/hero.html" $ }}
```

Comment:
We need to push the data we want into the partial. The below should work to push all params of a fragment into the right partial.
`{{ partial "fragments/hero.html" .Params }}`